go/exocom/src/client_registry/client_registry.go

```diff
@@ -0,0 +1,77 @@
+package clientRegistry
+
+import "encoding/json"
+
+// Client is the combination of a client name, service type and internal namespace
+type Client struct {
+	ClientName        string
```

Comment:
Didn't we name this `role`? The word "name" is too generic. There are three names here:
- service type: the "class" of the service
- instance name (coming soon): the name of the particular service instance ("user service instance #3")
- role: the role that the service instance plays inside the application (a MongoService instance could play the role UserService). This is only different when using message translation.